app/controllers/api/v1/ingredients_controller.rb

```diff
@@ -110,6 +133,12 @@ def ingredient_params
     )
   end
 
+  def ingredient_measures_params
+    params.require(:ingredient).permit(ingredient_measures_attributes: [
+      :id, :name, :quantity, :primary, :_destroy
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstArrayElementIndentation: Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.